accounts/templates/accounts/participant_list.html

```diff
@@ -44,16 +52,16 @@
    <div class="row mt-xl visible-md visible-lg visible-xl">
        <div class="col-md-3 col-md-offset-1">
            <strong> Username </strong>
-           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
-           <a aria-label="Sort descending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='-given_name' %}"><i aria-hidden="true" class="fa fa-chevron-down" role="button" name="name"></i></a>
+           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request page='1' sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
```

Comment:
Maybe a better explanation, passing in the page='1' is an override.  The query transform tag I wrote allows you to build a URL using whatever the current query params are, plus any new query params you pass in.   So passing in page=1 allows me to override the page, forcing 1 instead of the current page. (And sometimes I want the current page, like when I'm actively paginating).